packages/discovery-react-components/src/components/DocumentPreview/components/Highlight/typeUtils.ts

```diff
@@ -0,0 +1,10 @@
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+import { QueryResultPassage, QueryTableResult } from 'ibm-watson/discovery/v2';
+
+export function isPassage(obj: any): obj is QueryResultPassage {
+  return obj && 'passage_text' in obj;
+}
+
+export function isTable(highlight: any): highlight is QueryTableResult {
+  return !!highlight?.table?.location;
+}
```

Comment:
Were these moved to prevent circular-dependency? If so, you may now want to check other parts of the code that do this code and replace with calls to `isPassage` or `isTable`.

Reply:
Yes, that was the reason. I looked for uses of the old functions, but I'll also look for code that seems to be checking these same values. :+1: